ftw/copymovepatches/tests/test_move_updates_date_date.py

```diff
@@ -0,0 +1,11 @@
+from ftw.copymovepatches.tests import test_move_updates_modified_date
+from DateTime import DateTime
+
+
+class TestMoveUpdatesDateDate(test_move_updates_modified_date.TestMoveUpdatesModifiedDate):
```

Comment:
having issues with test discovery running tests multiple times? Introducing an inner base class as suggested in https://stackoverflow.com/questions/1323455/python-unit-test-with-base-and-sub-class may help here and get rid of the import trickery.

Reply:
Yes indeed, this is a test discovery hack. I did find this one not too bothering though. But of course I can also make a baseclass. Or simply put both classes in the same module.

Reply:
Also it is actually a pretty similar solution to having an inner baseclass, as it basically uses the fact that, and I quote here: "unittest only runs module-level classes that inherit from TestCase.".

Reply:
ok, fine 😏.